subschemas/substrate-chain/src/typeDefs/technicalCommittee.ts

```diff
@@ -0,0 +1,12 @@
+export default /* GraphQL */ `
+  type TechnicalCommitteeSummary {
+    memberCount: Int!
+    activeProposal: Int!
+    totalProposal: String!
```

Comment:
I would name this `activeProposals` or `activePoposalCount` 
Also `totalProposals` or `totalProposalCount` 

It just feels more representative and aligned.

Reply:
Sounds perfect!